.gitignore

```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 .next
+next-env.d.ts
```

Comment:
@hsher hello, `next-env.d.ts` should't be git-ignored, you need to include it in the repository, since it contains Next.js typing info